integration_tests/src/contract_test.rs

```diff
@@ -106,63 +112,53 @@ pub async fn validate_contract_exit_functionality(db_addr: Address) {
 
     assert!(res.is_err());
 
-    let res = add_exit_to_exit_list(
+    let _res = add_exit_to_exit_list(
         &contact,
         exit_1.clone(),
         db_addr,
         miner_private_key,
-        None,
+        Some(TX_TIMEOUT),
         vec![],
     )
     .await
     .unwrap();
 
-    contact
-        .wait_for_transaction(res, TX_TIMEOUT, None)
-        .await
-        .unwrap();
-
-    let res = add_exit_to_exit_list(
+    let _res = add_exit_to_exit_list(
         &contact,
         exit_2.clone(),
         db_addr,
         miner_private_key,
-        None,
+        Some(TX_TIMEOUT),
         vec![],
     )
     .await
     .unwrap();
 
-    contact
-        .wait_for_transaction(res, TX_TIMEOUT, None)
-        .await
-        .unwrap();
-
     let res = get_client_exit_list(&contact, miner_pub_key, db_addr)
         .await
         .unwrap();
 
+    println!("res is {:?}", res);
+
     assert_eq!(res, vec![exit_1.clone(), exit_2.clone()]);
 
-    let res = add_exit_to_exit_list(
+    let _res = add_exit_to_exit_list(
         &contact,
         exit_3.clone(),
         db_addr,
         miner_private_key,
-        None,
+        Some(TX_TIMEOUT),
         vec![],
     )
     .await
     .unwrap();
-    contact
-        .wait_for_transaction(res, TX_TIMEOUT, None)
```

Comment:
why remove this?

Reply:
The function above this has a parameter for timeout, that I added Some(TX_TIMEOUT) which does the waiting before returning. I added that parameter and removed the wait_for_transaction to remove redundant code